portal-impl/src/com/liferay/portal/internal/servlet/MainServlet.java

```diff
@@ -341,11 +341,16 @@ public void init() throws ServletException {
 		}
 
 		try {
-			String xml = StreamUtil.toString(
+			String shieldedContainerWebXml = StreamUtil.toString(
```

Comment:
@jorgediaz-lr Please inline variables that are only used once unless SF tells you not to:
- `shieldedContainerWebXml`
- `webXml`
- `root` in `_checkShieldedContainerWebXml()` method